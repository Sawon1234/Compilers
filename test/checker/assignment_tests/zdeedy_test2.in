class Fib (v : Int) {

    fun fib(n : Int) : Int {
        if ( !n ) {
            return 1; 
        }
    }
}

fun foo(y : Integer) : Integer = y * 2;

return 5;