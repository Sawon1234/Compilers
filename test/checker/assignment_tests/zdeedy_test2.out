
class Fib (v : Int) {
    fun fib(n : Int) : Int {
        if ( !n ) {
            return 1; 
        }
    }
}
